### DISCUSSION TITLE
PK-sim

### DISCUSSION THREAD

**Discussion by hong77777** [Open-Systems-Pharmacology/Forum#1019] [Questions & Problems] [answered]
What is the difference between these two P-gps?
![图片](https://user-images.githubusercontent.com/94173897/141600439-73034a37-8b35-429b-8428-9afe22c32c08.png)

**Comment by tobiasK2001** [comment, marked as answer]
Dear Hong,

you can click on the hyperlink for the Gene ID: 
![image](https://user-images.githubusercontent.com/25585248/141738935-c998a7f4-dc06-4de8-a262-93afb0d27db1.png)
Than it will take you to the respective GenBank entry, where you can check it all out.

Best, Tobias

**Reply by hong77777** [reply]
OK, I got it. Thank you very much,Tobias.
